Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM inclusivedesign/nodejs:4.2.1
```

Comment:
This will have to be updated periodically, but I think this should be 
    FROM inclusivedesign/nodejs:4.2.4
because 4.2.1 is on longer available. 

https://hub.docker.com/r/inclusivedesign/nodejs/tags/